<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectFilterSettings</class>
 <widget class="QWidget" name="ProjectFilterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Configure which files and folders inside the project folder should be included or excluded:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="filters"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="add">
         <property name="text">
          <string>Add</string>
         </property>
         <property name="icon">
          <iconset theme="list-add">
           <normaloff/>
          </iconset>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="remove">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveUp">
         <property name="text">
          <string>Move Up</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-up">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveDown">
         <property name="text">
          <string>Move Down</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-down">
           <normaloff/>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="KMessageWidget" name="messageWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header location="global">kmessagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
